flask_potion/__init__.py

```diff
@@ -182,8 +193,7 @@ def add_route(self, route, resource, endpoint=None, decorator=None):
 
         view = self.output(view_func)
```

Comment:
Simply you say? :)
I'm having an issue here. I either find a way to defer this so that I can access app configuration when the app is initialized (here app is the Blueprint and current_app laments that we're out of application context) or you tell me how to access the configs.